e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js

```diff
@@ -199,4 +200,48 @@ test.describe('Basic Condition Set Use', () => {
         const numOfUnnamedConditions = await page.locator('text=Unnamed Condition').count();
         expect(numOfUnnamedConditions).toEqual(1);
     });
+    test('ConditionSet should display appropriate view options', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5924'
+        });
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Alpha Sine Wave Generator"
+        });
+        await createDomainObjectWithDefaults(page, {
+            type: 'Sine Wave Generator',
+            name: "Beta Sine Wave Generator"
+        });
+        await createDomainObjectWithDefaults(page, {
+            type: 'Condition Set',
+            name: "Test Condition Set"
+        });
+
+        // Change the object to edit mode
+        await page.locator('[title="Edit"]').click();
+
+        // Expand the 'My Items' folder in the left tree
+        await page.locator('.c-tree__item__view-control.c-disclosure-triangle').first().click();
```

Comment:
This is fine locally but might fail in our testing environment where YAMCS dictionaries appear before My Items.

One strategy to get around this without requiring the name of the "My Items" folder is to navigate to one of the created Condition Sets, and then clicking the "Show selected item in tree" target button on the tree:

```js
await page.goto(conditionSet1.url);
await page.click('button[title="Show selected item in tree"]');
```